src/OneExplorer/ConfigObject.ts

```diff
@@ -0,0 +1,215 @@
+   *
+   */
+  private static importIni(filePath: string): object|null {
+    let configRaw: string;
+    try {
+      configRaw = fs.readFileSync(filePath, 'utf-8');
+    } catch (e) {
+      console.error(e);
+      return null;
+    }
+
+    // TODO check if toString() is required
+    return ini.parse(configRaw.toString());
+  }
+
+  /**
+   * Parse base models written in the ini object and return the absolute path.
+   *
+   * @param uri cfg uri is required to calculate absolute path
+   * ABOUT MULTIPLE BASE MODELS
+   * onecc doesn't support multiple base models.
+   * However, OneExplorer will show the config node below multiple base models
+   * to prevent a case that users cannot find their faulty config files on ONE explorer.
+   */
+  private static parseBaseModels = (uri: vscode.Uri, iniObj: object): vscode.Uri[] => {
+    let baseModels: string[] = [];
+    for (const ext of ['tflite', 'pb', 'onnx']) {
+      let confSection = iniObj[`one-import-${ext}` as keyof typeof iniObj];
+      let confKey = confSection ?.['input_path' as keyof typeof iniObj] as string;
+      if (confKey) {
+        baseModels.push(confKey);
+      }
+    }
+
+    // Get absolute paths by calculating from cfg file
+    // '..' or '//' are resolved here
+    baseModels = baseModels!.map(relpath => path.join(path.dirname(uri.fsPath), relpath));
+
+    // Remove duplicated entries
+    baseModels = [...new Set(baseModels)];
+
+    if (baseModels.length > 1) {
+      // TODO Notify the error with a better UX
+      // EX. put question mark next to the config icon
+      console.warn(`Warning: There are multiple input models in the configuration. (path: ${uri})`);
+    }
+
+    if (baseModels.length === 0) {
+      // TODO Notify the error with a better UX
+      // EX. showing orphan nodes somewhere
+      console.warn(`Warning: There is no input model in the configuration. (path: ${uri})`);
+    }
+
+    // Return as list of uri
+    return baseModels.map(abspath => vscode.Uri.file(abspath));
+  };
+
+  /**
+   * Find derived models written in the ini object and return the absolute path.
+   *
+   * @param uri cfg uri is required to calculate absolute path
+   */
+  private static parseDerivedModels = (uri: vscode.Uri, iniObj: object): vscode.Uri[] => {
+    const derivedModelLocator = [
+      {section: 'one-import-tf', key: 'output_path'},
+      {section: 'one-import-tflite', key: 'output_path'},
+      {section: 'one-import-onnx', key: 'output_path'},
+      {section: 'one-import-bcq', key: 'output_path'},
+      {section: 'one-optimize', key: 'input_path'},
+      {section: 'one-optimize', key: 'output_path'},
+      {section: 'one-quantize', key: 'input_path'},
+      {section: 'one-quantize', key: 'output_path'},
+      {
+        section: 'one-codegen',
+        key: 'command',
+        filt: (str: string): string[] => {
+          return str.split(' ').filter(
+              e => path.extname(e) === '.tvn' || path.extname(e) === '.circle');
+        }
```

Comment:
Can you please add todo to apply  `Executor.getExecutableExt()` method here? If you don't mind, I'll update after some days.

Reply:
Thanks for the suggestion. I will apply them. (I think they are many)

Reply:
Well I checked the code but it will not only contain executable exts, I will also involve those all derived models. I will add `// Get ext lists from backend` instead. :-D

Reply:
Oh, Thank you for your check. 🙇 That seems a better comment.